scraper/src/github-scraper/parseEvents.ts

```diff
@@ -9,7 +9,19 @@ import { parseISO } from "date-fns";
 import { isBlacklisted } from "./utils.js";
 import { octokit } from "./config.js";
 const processedData: ProcessData = {};
-
+async function fetchdefaultbranch(org: string, repo: string) {
+  try {
+    const { data } = await octokit.request('GET /repos/{owner}/{repo}', {
+      owner: org,
+      repo: repo,
+    });
+    return data.default_branch;
+  } catch (e) {
+    console.error(
+      `Error fetching default branch for organisation ${org} /${repo} `,
+    );
+  }
+}
```

Comment:
Can we maintain a cache of the results? So that we don't need to keep hitting GitHub's APIs for every PRs that collaboration event is to be added?